src/entity.js

```diff
@@ -780,25 +782,17 @@ Entity.setup = function(options) {
       var start = process.hrtime();
       return method.apply(client, arguments).then(function(result) {
         var d = process.hrtime(start);
-        reporter({
-          component:    options.component,
-          process:      options.process,
-          duration:     d[0] * 1000 + (d[1] / 1000000),
-          table:        options.table,
-          method:       name,
-          error:        'false'
-        });
+        if (reporter) {
+          reporter.measure(name + '.success', d[0] * 1000 + (d[1] / 1000000));
+          reporter.count(name + '.success');
```

Comment:
How about `options.monitor.count([name, 'success']);` just to abstract the separator away..
Still nit...